### PR TITLE
Initial commit for Rule 86 with ArangoDB backend.

### PR DESCRIPTION
**Pull request by johanfol**

### CODE REVIEW

File: rule-86-arangodb/jest.config.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
+'use strict';
```
Comment (y7usuf) [inline rule-86-arangodb/jest.config.js:1]: We need to remove that file.

Comment (y7usuf) [inline rule-86-arangodb/jest.config.js:1]: We need to remove that file.

Comment (y7usuf) [reply rule-86-arangodb/jest.config.js:1]: .map file as well

**Review by JohannWNel — APPROVED** [review]

File: rule-86-arangodb/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+  app.listen(configuration.hostPort, () => {
+    console.log(
+      `Predicate Builder Service - Listening on: http://localhost:${configuration.hostPort}`,
+    );
+  });
+};
+start();
+
+/* //LEGACY STARTUP - KAFKA ETC
```
Comment (y7usuf) [inline rule-86-arangodb/src/index.ts:33]: Do we need to keep legacy code in comment here?

**Review by y7usuf — APPROVED** [review]
It looks good 👍 I'm happy to approve, but I just made some comments.

**Review by y7usuf — APPROVED** [review]

**Pull request merged**